bin/assigner/src/main.cpp

```diff
@@ -858,6 +929,12 @@ int main(int argc, char *argv[]) {
         "Or use \"-f hex\", hex output format is also supported now\n"
     );
 
+    std::vector<std::size_t> column_sizes = {};
```

Comment:
may be better to init from default `ParametersPolicy` values here, and update if provided command line option, or set default under `else`